CardDictionaries/ArcaneRising/ARCWizard.php

```diff
@@ -595,11 +598,30 @@ function ProcessSurge($cardID, $player, $target)
         break;
       case "ROS189": case "ROS190": case "ROS191":
         WriteLog("Surge active, returning a sigil from graveyard to hand");
-        MZMoveCard($player, "MYDISCARD:isAuraSigil", "MYHAND", may:true);
+        MZMoveCard($player, "MYDISCARD:subtype=Aura;nameIncludes=Sigil", "MYHAND", may:true);
         break;
       case "ROS198": case "ROS199": case "ROS200":
         WriteLog("Surge active, gainting 2 resources");
         GainResources($player, 2);
+      case "ROS201": case "ROS202": case "ROS203": //perennial aetherbloom
+        WriteLog("Surge active, returning to the bottom of the deck");
+        AddBottomDeck($cardID, $player, "STACK"); //create a copy on the bottom
+        $discard = &GetDiscard($player);
+        array_pop($discard); array_pop($discard); //it will always be the top card in discard
+      case "ROS173": case "ROS174": case "ROS175":
+        WriteLog("Surge Active, returning sigils to the deck");
+        $auras = &GetAuras($player);
+        for ($i = count($auras) - AuraPieces(); $i >= 0; $i -= AuraPieces()){
+          WriteLog($i . "-" . $auras[$i]);
+          $auraName = CardName($auras[$i]);
+          if (DelimStringContains($auraName, "Sigil", partial:true)){
+            AddBottomDeck($auras[$i], $player, "STACK");
+            for ($j = 0; $j < AuraPieces(); $j++){
+              unset($auras[$i+$j]);
+            }
+          }
+        }
+        AddDecisionQueue("SHUFFLEDECK", $player, "-");
```

Comment:
not really anything to change but I had no idea this was the standard for shuffling the deck. it's amusing to me.